src/makeRobot.js

```diff
@@ -38,7 +38,64 @@
  * @return {object}
  */
 function makeRobot(name, wheels, version) {
-  // write code here
+  const newRobot = {
+    'name': name,
+    'wheels': wheels,
+    'version': version,
```

Comment:
If the key and value have the same name, then it can be written as one word

```suggestion
    name,
    wheels,
    version,
```